include/multipass/logging/log.h

```diff
@@ -77,15 +85,15 @@ log(Level level, const std::string& category, fmt::format_string<Arg0, Args...>
  * @param [in] args Format arguments, if any
  */
 template <Level level, typename... Args>
-constexpr void log(const std::string& category, fmt::format_string<Args...> fmt, Args&&... args)
+constexpr void log(std::string_view category, fmt::format_string<Args...> fmt, Args&&... args)
 {
     if constexpr (sizeof...(Args) > 0)
     {
         // Needs formatting
         logging::log(level, category, fmt, std::forward<Args>(args)...);
         return;
     }
-    logging::log(level, category, CString{fmt.get().data()});
+    logging::log(level, category, std::string_view{fmt.get().data(), fmt.get().size()});
```

Comment:
It's a shame that fmt::string_view is not directly convertible to std::string_view.